Sources/Extraction/Filters/HillOrientation.h

```diff
@@ -8,27 +8,10 @@
 #include "General/PointF.h"
 #include "General/BlockMap.h"
 #include "General/BinaryMap.h"
+#include "General/List.h"
 
-typedef struct NeighborInfo NeighborInfo;
-typedef struct HillOrientation HillOrientation;
+UInt16Array2D HillOrientation_Detect(FloatArray2D image, Size imageDimensions, BinaryMap * blockMask, BlockMap * blocks);
+BoolArray2D HillOrientation_BlockMapToPixelMask(Size imageDimensions, BinaryMap * blockMask, BlockMap * blocks);
```

Comment:
Do we really need to expose this for everyone? I pressume that you are using it for testing?